circuits/cpp/src/aztec3/circuits/rollup/base/.test.cpp

```diff
@@ -658,7 +658,9 @@ TEST_F(base_rollup_tests, native_compute_membership_historic_private_data_negati
         aztec3::circuits::rollup::native_base_rollup::base_rollup_circuit(builder, inputs);
 
     ASSERT_TRUE(builder.failed());
-    ASSERT_EQ(builder.get_first_failure().message, "Membership check failed: historic private data tree roots 0");
+    ASSERT_EQ(builder.get_first_failure().message,
+              "Membership check failed: base_rollup_circuit: historical root in rollup constants not in historic "
+              "private data tree roots at kernel index 0");
```

Comment:
what is kernel index 0?

Reply:
first kernel (out of the two in a base). You have a better way to say it?